BrowserSelect/Form1.cs

```diff
@@ -104,9 +105,14 @@ public void add_rule(Browser b)
             }
             else
             {
-                // in case pattern was not ambiguous, just set the pattern as rule and open the url
-                var pat = (_alwaysRule.mode == 1) ? _alwaysRule.tld_rule : _alwaysRule.second_rule;
-                add_rule(b, pat);
+
+                _alwaysAsk = new ContextMenu((new[] { _alwaysRule.tld_rule, _alwaysRule.second_rule, _alwaysRule.url_rule })
```

Comment:
Wouldn't this show invalid rules (e.g. *.com) or duplicated options if there was no ambiguity? Maybe keep the check and hide tld_rule or second_rule only showing the other one?
![image](https://user-images.githubusercontent.com/4184939/86516823-f836ac00-be38-11ea-9004-a9461052d98e.png)